ui/components/multichain/app-header/app-header.js

```diff
@@ -195,15 +193,12 @@ export const AppHeader = ({ location }) => {
               {popupStatus ? (
                 <Box className="multichain-app-header__contents__container">
                   <Tooltip title={currentNetwork?.nickname} position="right">
-                    <AvatarNetwork
+                    <PickerNetwork
                       className="multichain-app-header__contents--avatar-network"
                       ref={menuRef}
                       as="button"
-                      aria-label={t('networkMenu')}
```

Comment:
I think we still need this aria-label as there is no text describing what the button does for screen readers?

Reply:
Yup! Updated https://github.com/MetaMask/metamask-extension/pull/19514/commits/84530cab2ebdf1edebbfae3bc1719b32efc42e63